recipe/meta.yaml

```diff
@@ -1,20 +1,18 @@
 {% set name = "plyvel" %}
-{% set version = "1.4.0" %}
+{% set version = "1.5.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 4ea98bea04ebf0f44747bacdfafefc8827787106fbb787f0aedc46482b2dfd53
+  sha256: cd918e0b31690abcd3d202a8742caf951ab2fe1573de7af71c38456847f9202b
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  # trigger: 0
-  ignore_run_exports:  # [win]
-    - leveldb          # [win]
+  skip: True  # [py<37]
```

Comment:
I don't see restriction on the python version upstream. Where did you find it?

Reply:
I went based on this: https://github.com/wbolster/plyvel/blob/1.5.0/tox.ini#L2

Reply:
I'm not an expert on `tox`, but I think that's just the environments they are testing the package in and does not indicate that it's incompatible with other versions.

Reply:
That's fair. I've updated to skip <=3.5 based on prior release notes: https://plyvel.readthedocs.io/en/latest/news.html#plyvel-1-3-0